bin/main.ml

```diff
@@ -224,7 +227,9 @@ let common =
          orig)
         x
         display
+        build_dir
     =
+    let build_dir = Option.value ~default:"_build" build_dir in
```

Comment:
Why not put the default value in the `Arg.(value ...)`?

Reply:
I think it's because the default would have to be duplciated for bootstrap, but I'm no longer sure. I agree that making the default part of the CLI is correct.